docs/api-graduation-guidelines.md

```diff
@@ -0,0 +1,31 @@
+This document defines the expectations of how API graduation is handled across the github.com/kubevirt organization. Individual projects within the KubeVirt org may decide to append additional criteria and timeline constraints to the items outlined here, but should not deviate from the broad expectations as defined in this document.
+
+# Graduation Phase Expecations
+
+## Alpha
+* Feature incomplete, meaning the API might be half implemented as functionality lands in phases.
+* Basic testing exists proving that the merged functionality works e2e.
+* No backwards compatibility guarantees meaning API can mutate and be deprecated without warning.
+* Available for early adopters to test and provide initial feedback.
+
+## Beta
+* User guide documentation exists.
+* Has early adopter commitment, meaning at least one end user or vendor has provided feedback that the feature meets their needs and will be adopted. 
+* Well tested end to end including update tests that guarantee forward compatibility.
+* Stable API, meaning no backwards incompatible API changes will occur.
+* Support for the API will not be dropped without a lengthy deprecation phase.
+
+## General Availability
+* Core functionality is feature complete, meaning the API isn't lacking any critical functionality.
+* Proven in production use cases.
```

Comment:
Do we expect a v1 stable API at this point?

Reply:
yes, GA would mean `v1`